Worlds/FelipeWorld/0.tscn

```diff
@@ -0,0 +1,8 @@
+[gd_scene load_steps=2 format=4 uid="uid://ck0cup6ysca3q"]
+
+[ext_resource type="TileSet" uid="uid://caem805hxljgn" path="res://CandyWrapper/TileSet.tres" id="1_tcuux"]
+
+[node name="TileMapLayer" type="TileMapLayer"]
+use_parent_material = true
+tile_map_data = PackedByteArray("AAD/////AwAAAAAAAAAAAP//AwAAAAAAAAABAP//AwAAAAAAAAACAP//AwAAAAAAAAADAP//AwAAAAAAAAAEAP//AwAAAAAAAAAFAP//AwAAAAAAAAAGAP//AwAAAAAAAAAHAP//AwAAAAAAAAAIAP//AwAAAAAAAAAJAP//AwAAAAAAAAAKAP//AwAAAAAAAAALAP//AwAAAAAAAAAMAP//AwAAAAAAAAANAP//AwAAAAAAAAAOAP//AwAAAAAAAAAPAP//AwAAAAAAAAAQAP//AwAAAAAAAAARAP//AwAAAAAAAAASAP//AwAAAAAAAAD//wAAAwAAAAAAAAASAAAAAwAAAAAAAAD//wEAAwAAAAAAAAASAAEAAwAAAAAAAAD//wIAAwAAAAAAAAASAAIAAwAAAAAAAAD//wMAAwAAAAAAAAASAAMAAwAAAAAAAAD//wQAAwAAAAAAAAASAAQAAwAAAAAAAAD//wUAAwAAAAAAAAASAAUAAwAAAAAAAAD//wYAAwAAAAAAAAASAAYAAwAAAAAAAAD//wcAAwAAAAAAAAD//wgAAwAAAAAAAAASAAgAAwAAAAAAAAD//wkAAwAAAAAAAAD//woAAwAAAAAAAAD//wsAAwAAAAAAAAD//wwAAwAAAAAAAAD//w0AAAAAAAAAAAD//w4AAwAAAAAAAAD//w8AAwAAAAAAAAD//xAAAwAAAAAAAAD//xEAAwAAAAAAAAD//xIAAwAAAAAAAAAAAA0AAAAAAAAAAAAAABIAAwAAAAAAAAABAA0AAAAAAAAAAAABABIAAwAAAAAAAAACAA0AAAAAAAAAAAACABIAAwAAAAAAAAADAA0AAAAAAAAAAAADABIAAwAAAAAAAAAEABIAAwAAAAAAAAAFAA0AAAAAAAAAAAAFABIAAwAAAAAAAAAGAA0AAAAAAAAAAAAGABIAAwAAAAAAAAAHABIAAwAAAAAAAAAIABIAAwAAAAAAAAAJABIAAwAAAAAAAAAKABIAAwAAAAAAAAALABIAAwAAAAAAAAAMABIAAwAAAAAAAAANABIAAwAAAAAAAAAOABIAAwAAAAAAAAAPABIAAwAAAAAAAAAQABIAAwAAAAAAAAARABIAAwAAAAAAAAASAAkAAwAAAAAAAAASAAoAAwAAAAAAAAASAAsAAwAAAAAAAAASAA4AAwAAAAAAAAASAA8AAwAAAAAAAAASABAAAwAAAAAAAAASABEAAwAAAAAAAAASABIAAwAAAAAAAAAMAA0AAAAAAAAAAAAPAA0AAAAAAAAAAAAQAA0AAAAAAAAAAAARAA0AAAAAAAAAAAASAAwAAwAAAAAAAAASAA0AAAAAAAAAAAAIAA0AAAAAAAAAAAAJAA0AAAAAAAAAAAALAA0AAAAAAAAAAAAAAAwAAQAAAAAAAAASAAcAAwAAAAAAAAAEAA0AAAAAAAAAAAAHAA0AAAAAAAAAAAAKAA0AAAAAAAAAAAANAA0AAAAAAAAAAAAOAA0AAAAAAAAAAAAAAAcAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAkAAAAAAAAAAAAAAAoAAAAAAAAAAAAAAAsAAAAAAAAAAAABAAcAAAAAAAAAAAABAAkAAAAAAAAAAAADAAsAAAAAAAAAAAAEAAsAAAAAAAAAAAADAAcAAAAAAAAAAAADAAgAAAAAAAAAAAADAAkAAAAAAAAAAAADAAoAAAAAAAAAAAAEAAcAAAAAAAAAAAAEAAkAAAAAAAAAAAAGAAcAAAAAAAAAAAAGAAgAAAAAAAAAAAAGAAkAAAAAAAAAAAAGAAoAAAAAAAAAAAAGAAsAAAAAAAAAAAAHAAsAAAAAAAAAAAAJAAcAAAAAAAAAAAAJAAgAAAAAAAAAAAAJAAkAAAAAAAAAAAAJAAoAAAAAAAAAAAAJAAsAAAAAAAAAAAALAAcAAAAAAAAAAAALAAgAAAAAAAAAAAALAAkAAAAAAAAAAAALAAoAAAAAAAAAAAALAAsAAAAAAAAAAAAMAAcAAAAAAAAAAAAMAAkAAAAAAAAAAAANAAgAAAAAAAAAAAAPAAcAAAAAAAAAAAAPAAgAAAAAAAAAAAAPAAkAAAAAAAAAAAAPAAoAAAAAAAAAAAAPAAsAAAAAAAAAAAAQAAcAAAAAAAAAAAAQAAkAAAAAAAAAAAAQAAsAAAAAAAAAAAA=")
```

Comment:
It's hard to see it here 😁 but you have a player in this tilemap. The first (and last) levels are treated specially, with the player being added at a hardcoded location (and not responding to input). So the effect is that the title screen has two copies of the player, one of which can be moved left and right. It's harmless and I expect that in due course that special case in the game logic will be removed. Up to you if you want to leave it like this, or remove the player from the tilemap.